rel/relx.config.src

```diff
@@ -9,6 +9,7 @@ Filter = ['rabbitmq_codegen'
           ,'.settings'
           ,'skel'
           ,'parse_trans'
+          ,'sup'
```

Comment:
@lazedo, this was unintended, this commit just slipped in pull request from my working branch.